src/sidebar/util/annotation-sharing.js

```diff
@@ -1,47 +1,61 @@
 /**
+ * @typedef {import('../../types/api').Annotation} Annotation
  * @typedef {import('../../types/config').HostConfig} HostConfig
  */
 
 import serviceConfig from '../service-config';
 
 /**
- * Is the sharing of annotations enabled? Check for any defined `serviceConfig`,
- * but default to `true` if none found.
+ * Generate a URI for sharing: a bouncer link built to share annotations in
+ * a specific group (groupID) on a specific document (documentURI).
  *
- * @param {HostConfig} settings
+ * @param {string} documentURI
+ * @param {string} groupID
+ * @return {string}
+ */
+export function getSharingLink(documentURI, groupID) {
```

Comment:
This function's logic was moved from the `ShareAnnotationsPanel` component to here.